src/analyze/analyze-verify.h

```diff
@@ -6,5 +6,17 @@
 #include "execute.h"
 #include "path-lookup.h"
 
+typedef enum ReturnErrorOn {
```

Comment:
I think this should be called FailVerificationOn or so

Reply:
This was changed as such following feedback - @keszybz are you OK with `FailVerificationOn` instead of `ReturnErrorOn`?

Reply:
Yeah, I think 'FailVerificationOn' is better.